code/game/jobs/faction/faction.dm

```diff
@@ -45,7 +45,7 @@
 				role.blacklisted_species = J.blacklisted_species
 			. += role
 
-/datum/faction/proc/get_selection_error(datum/preferences/prefs)
+/datum/faction/proc/get_selection_error(datum/preferences/prefs, var/user=usr)
```

Comment:
why not just use `usr` instead?

Reply:
Because it should be explicitly specified what user you are evaluating.